hugo/content/sponsors/index.md

```diff
@@ -35,7 +35,7 @@ Also, we haven’t experienced a single downtime in many years.
 
 {{< sponsors/logosvg "logos/macstadium.svg" "MacStadium’s logo" >}}
 
-Thanks to [MacStadium](https://www.macstadium.com) we can maintain support for macOS, as they provide us access to a 2012 Mac Mini.
+Thanks to [MacStadium](https://www.macstadium.com) we can maintain support for macOS, as they provide us access to a recent Mac Mini.
```

Comment:
```suggestion
Thanks to [MacStadium](https://www.macstadium.com) we can maintain support for macOS. They provide us access to a Mac Mini.
```

Shorter sentences are better.

I don't think it's obvious what recent refers to. (recently? recent version? hardware? os/software version?

Omitting the *recent* label is the simplest solution. It still holds the information we want to convey.